fiftyone/utils/utils3d.py

```
@@ -1065,6 +1063,31 @@ def _visit_node_dfs(node):
     return pcd_path
 
 
+def _read_point_cloud(filepath):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_read_point_cloud`

(ANN202)